README.md

```diff
@@ -1,3 +1,4 @@
+````
```

Comment:
What this 4 backticks for, it breaks README
![image](https://github.com/creatorsgarten/kaogeek-discord-bot/assets/1791353/93f39461-4b65-48ee-a69b-378f99c49a5a)